src/QSymbolicsBase/predefined_fock.jl

```diff
@@ -26,6 +26,21 @@ end
 SqueezedState(z::Number) = SqueezedState(z, inf_fock_basis)
 symbollabel(x::SqueezedState) = "0,$(x.z)"
 
+"""Two-mode squeezed vacuum state, or EPR state, in defined Fock basis."""
+@withmetadata struct EPRState <: SpecialKet
+    z::Number
+    basis::CompositeBasis
+    function EPRState(z::Number, basis::CompositeBasis)
+        bases = basis.bases
+        length(bases) == 2 && all(x -> x isa FockBasis, bases) || 
+                throw(ArgumentError(lazy"The underlying basis for an EPR state must be a
+                                        tensor product of two bases for single-mode quantum systems."))
+        return new(z, basis)
+    end
+end
+EPRState(z::Number) = EPRState(z, inf_fock_basis^2)
```

Comment:
I am only now noticing, but `SpecialKet` might be a bit of a broken implementation. This and other `SpecialKet` subtypes are marked as `isexpr()=false` while they are definitely expressions, not leaf nodes. No need to fix it in this PR, but this is an issue we should probably track.